care/emr/tests/test_organization_api.py

```
@@ -0,0 +1,998 @@
+        self.client.force_authenticate(user=otp_user)
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 200)
+
+        org_types = [org["org_type"] for org in response.data["results"]]
+        self.assertTrue(all(org_type == "govt" for org_type in org_types))
+        self.assertNotIn("team", org_types)
+
+    def test_filter_organizations_by_parent(self):
+        """Test that organizations can be filtered by parent."""
+        self.client.force_authenticate(user=self.super_user)
+        parent_org = self.create_organization(
```

Ruff (0.12.2)

Local variable `parent_org` is assigned to but never used

Remove assignment to unused variable `parent_org`

(F841)